### ISSUE TITLE
Internet Explorer 11 cannot Resolve next page correctly!

### ISSUE THREAD

**Issue by siaimes** [theme-next/hexo-theme-next#30]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
IE show my web site correctly！

### Actual behavior 
sidebar、TOC and gitment comment system cannot display.

### Steps to reproduce the behavior 
1. open Chrome and entering [https://blog.siaimes.me](https://blog.siaimes.me), all things I want are appear.
2. open Internet Explorer  entering [https://blog.siaimes.me](https://blog.siaimes.me), sidebar、TOC and gitment comment system disappear.

* Link to demo site with this issue: [https://blog.siaimes.me](https://blog.siaimes.me)
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
v8.9.3
5.5.1
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.4.2"
  },
  "dependencies": {
    "hexo": "^3.2.0",
    "hexo-deployer-git": "^0.3.1",
    "hexo-deployer-rsync": "^0.1.3",
    "hexo-generator-archive": "^0.1.4",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index": "^0.2.0",
    "hexo-generator-search": "^2.1.1",
    "hexo-generator-sitemap": "^1.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.0",
    "hexo-renderer-pandoc": "^0.2.0",
    "hexo-renderer-stylus": "^0.3.1",
    "hexo-server": "^0.2.2"
  }
}
```

#### Hexo version

```
hexo: 3.4.2
hexo-cli: 1.0.4
os: Windows_NT 6.1.7601 win32 x64
http_parser: 2.7.0
node: 8.9.3
v8: 6.1.534.48
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2n
icu: 59.1
unicode: 9.0
cldr: 31.0.1
tz: 2017b
```

#### Hexo Configuration

```yml

```

### NexT Information

**NexT Version:**

```
[] Latest Master branch.
[x] Latest Release version.
[] Old version -
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[] Pisces
[x] Gemini
```

#### NexT Configuration:

```yml

```

**Comment by ivan-nginx** [comment]
![image](https://user-images.githubusercontent.com/16944225/34906390-c7275cc8-f87d-11e7-98a4-423b0e6977ce.png)
Old versions of IE (IE 11 too) [not support by Microsoft from January 12, 2016](https://www.microsoft.com/en-us/windowsforbusiness/end-of-ie-support). [IE 11 was supported till 2015 year](https://ru.wikipedia.org/wiki/Internet_Explorer_11) and for now it can't totally show all CSS3 styles (can't show shadow-borders in Gemini, transparents, etc.) and some JS features (sidebar in Pisces/Gemini, TOC, footer alignments).

Miscrosoft suggest to upgrade from IE to [Edge](https://en.wikipedia.org/wiki/Microsoft_Edge), and Edge already support all new features above.
![image](https://user-images.githubusercontent.com/16944225/34906375-72efca0a-f87d-11e7-9ecf-0dab3a7f33e0.png)

But if u still want to use IE, u can downdgrade NexT to version to 4.x or try to use any another simple theme with old css styles and without new JS features.

**Comment by siaimes** [comment]
@ivan-nginx Can you delete this Issue？

**Comment by ivan-nginx** [comment]
@siaimes yes, i can. No more issue here?

**Comment by ivan-nginx** [comment]
Sidebar fixed under IE11 in #498 pull.
